src/api/app/views/webui/request/show.html.erb

```diff
@@ -254,8 +254,8 @@
             <div class="review_descision_display <%= 'hidden' if index != 0 %>" id="review_descision_display_<%= index %>">
               <%= form_tag(action: 'modify_review') do %>
                   <p><%= user_with_realname_and_icon(review[:who], short: true) %> requested:</p>
-                  <p><%= simple_format(review_request_reason(@bs_request, review)) %></p>
-                  <p><%= text_area_tag('comment', review[:reason], size: '80x2', style: 'width: 99%', class: 'review_comment', placeholder: 'Please comment on your decision') %></p>
+                  <p><%= simple_format(review[:reason]) %></p>
```

Comment:
what we're missing here is the 'No reason given' that the old code had.

Reply:
But 'X requested: No reason given' is a little bit as in the joke 'MAMA, wake up! There was someone at the door, but I said him you are sleeping.'